superset/views/core.py

```diff
@@ -3127,8 +3127,17 @@ class CssTemplateAsyncModelView(CssTemplateModelView):
 @app.after_request
 def apply_http_headers(response):
     """Applies the configuration's http headers to all responses"""
-    for k, v in config.get("HTTP_HEADERS").items():
+
+    override_http_headers = config.get("OVERRIDE_HTTP_HEADERS", {})
```

Comment:
Given that `HTTP_HEADERS ` and `OVERRIDE_HTTP_HEADERS` are explicitly defined in the config I prefer not using `get(...)` as this can be implied that it's optional. In Python 3 line #3131 and #3133 could be reduced to:

```
override_http_headers = {**config["OVERRIDE_HTTP_HEADERS"], **config["HTTP_HEADERS"]}
```

Reply:
HTTP_HEADERS will no longer be explicitly defined, but i'll update the other one